indico/modules/users/templates/dashboard.html

```diff
@@ -82,6 +82,30 @@ <h3>{{ user.full_name }}</h3>
                         </a>
                     </div>
                 </div>
+                {% if unlisted_events %}
+                    <div class="dashboard-box">
+                        <h3>{% trans %}Your unlisted events{% endtrans %}</h3>
+                        <ul>
```

Comment:
I just realized that we use `<ol>` for the other boxes and have CSS rules that explicitly use `ol`. So let's either keep it (even though it's weird) or update it in the other places and css as well

![image](https://user-images.githubusercontent.com/179599/131324392-1dfda2ad-daae-4fa2-863d-82d6e894e66c.png)